x-pack/plugins/security_solution/public/management/cypress/support/data_loaders.ts

```diff
@@ -8,7 +8,10 @@
 // / <reference types="cypress" />
 
 import type { CasePostRequest } from '@kbn/cases-plugin/common/api';
-import { sendEndpointActionResponse } from '../../../../scripts/endpoint/agent_emulator/services/endpoint_response_actions';
+import {
+  sendEndpointActionResponse,
+  sendFleetActionResponse,
+} from '../../../../scripts/endpoint/agent_emulator/services/endpoint_response_actions';
```

Comment:
I suggest you don't use services from `agent_meulator` because they are private to that utility. I would propose you refactor the ones you need and move them to `script/endpoint/common`.